### PR TITLE
Add smoke and dust post-processing

### PR DESCRIPTION
**Pull request by EricJames-NOAA**
Adding the capability to output smoke and dust in RRFS runs.  This PR uses previous code used to output smoke in RAP / HRRR, but adds similar variables (hybrid / pressure level fields, vertically-integrated fields) for dust.  Also, a combined smoke and dust AOD is output as AOTK.  The code was tested on Hera using realtime files from RRFS-SD (NA_3km domain).  

One remaining issue to work out: how to distinguish smoke vs. dust in the GRIB2 files?  Currently both are encoded as MASSDEN.  We also have yet to add Fire Radiative Power.

### CODE REVIEW

**Comment by hu5970** [comment]
Any more changes needed for this PR ?

File: sorc/ncep_post.fd/INITPOST_NETCDF.f [hunk 1]
```diff
@@ -19,21 +19,23 @@
 !> 2022-04-08 | Bo Cui        | 2D decomposition for unified fv3 read interfaces
 !> 2022-06-05 | Hui-Ya Chuang | Modify dx/dy computation for RRFS domain over north pole
 !> 2022-07-10 | Wen Meng      | Output lat/lon on four coner points of rotated lat-lon grids in text file.
-!> 2022-07-18 | Wen Meng      | Read instant top of atmos ULWRF from model
+!> 2022-07-18 | Wen Meng      | Read instant top of atmos ULWRF from modela
```
Comment (hu5970) [inline sorc/ncep_post.fd/INITPOST_NETCDF.f]: last word should be "model"

**Comment by EricJames-NOAA** [comment]
@hu5970 the last change will be outputting smoke emissions in the GRIB2 file.  I should be able to get it tested this week.  Sorry for the delay.

**Comment by hu5970** [comment]
@EricJames-NOAA No problem. The code looks all good. We will try to merge this PR soon so we can have smoke and dust record in grib2 files when we deploy the baseline E code in real-time for NA 3km grid. Ming

**Comment by hu5970** [comment]
@EricJames-NOAA I see there are changes to the branch. Is this PR ready to be reviewed and merged. You may need to update or make a new PR to include you recent changes. Thanks, Ming

**Comment by EricJames-NOAA** [comment]
@hu5970 I'm still working on the details of the smoke / dust encoding, and also outputting smoke emissions.   What is our deadline for merging this PR?

**Comment by hu5970** [comment]
@EricJames-NOAA No worries.  I am working on deploy the smoke/dust for real-time CONUS 3km run and we should be able to get stable real-time runs next week. If you can make this PR work next week, that would be great. I think Ravan can check the NetCDF for now to make sure the smoke and dust looks OK. They also working on generating smoke emission files now. Thanks, Ming

**Comment by EricJames-NOAA** [comment]
@hu5970 the PR is ready now.  I tested it for NA_3km domain on Hera and CONUS_3km domain on Jet.  

I will also open a PR to regional_workflow to make the UPP static files consistent with this PR.

**Review by hu5970 — APPROVED** [review]

**Pull request merged**